cmd/leanix-k8s-connector/main.go

```diff
@@ -48,16 +48,13 @@ func main() {
 		log.Info("Failed to authenticate. Terminating..")
 	}
 	if viper.GetBool(utils.IrisFlag) {
-		log.Info("Enabled new Integration API")
+		log.Info("Enabled Iris")
 
-		irisScanner, err := iris.NewScanner(
+		irisScanner := iris.NewScanner(
```

Comment:
In general: The factory pattern would require an error as a second return value so you could react quickly

Here: It's fine to remove it since there was no possibility an error could be produced

Reply:
ok, thats good to know.